contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -113,69 +126,83 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
 
   constructor(
     address _admin,
-    IERC20Ext _rewardToken,
+    address[] memory _rewardTokens,
     IKyberRewardLocker _rewardLocker
   ) PermissionAdmin(_admin) {
-    rewardToken = _rewardToken;
+    rewardTokens = _rewardTokens;
     rewardLocker = _rewardLocker;
 
-    _rewardToken.safeApprove(address(_rewardLocker), type(uint256).max);
+    // approve allowance to reward locker
+    for(uint256 i = 0; i < _rewardTokens.length; i++) {
+      if (_rewardTokens[i] != address(0)) {
+        IERC20Ext(_rewardTokens[i]).safeApprove(address(_rewardLocker), type(uint256).max);
+      }
+    }
   }
 
   /**
    * @dev allow admin to withdraw only reward token
    */
-  function adminWithdraw(uint256 amount) external onlyAdmin {
-    rewardToken.safeTransfer(msg.sender, amount);
+  function adminWithdraw(uint256 rewardTokenIndex, uint256 amount) external onlyAdmin {
+    IERC20Ext rewardToken = IERC20Ext(rewardTokens[rewardTokenIndex]);
+    if (rewardToken == IERC20Ext(0)) {
```

Comment:
Without a `receive` method, I don't think this contract can receive native token (eth)